edge-functions/ab-testing-statsig/pages/_middleware.ts

```diff
@@ -0,0 +1,53 @@
+import { NextRequest, NextResponse } from 'next/server'
+import statsig from 'statsig-node'
+
+// Store a cookie for the user
+const UID_COOKIE = 'uid'
```

Comment:
The cookie should be set based on the feature flag, not the user id, `feature-flag-home` might be a good option.